lib/Settings/JobProfiles/tests/JobProfiles-test.js

```diff
@@ -25,6 +26,23 @@ import {
 
 const parentMutator = buildMutator();
 
+const JobProfilesContainer = ({
```

Comment:
The mentioned examples use functions to prevent some code duplications, but in the current (and mentioned by the links) case we have to create a component to use hooks inside of it. So please, do we still need to create a function just to wrap a mountWithContext call?

Reply:
Well, I think it should work too with function approach even if the underlying component is rendering hooks it, have you tried it?

Reply:
In general, yes, I tried some approaches but only the current provided the needed result. 
I'm not sure that I got your idea about underlying component. Please, could you clarify this point? Or, maybe, there are some examples and you can link to them.

Reply:
I see what you mean -  for hooks we need to use the one which you suggested. I think in that case for hooks like the approach we are good to use the one which is here and for other cases use the approach which I referred.

Reply:
Yes, sounds good.